### PR TITLE
Fix `show state`

### PR DESCRIPTION
**Pull request by jseagrave21**
**What current issue(s) does this address, or what feature is it adding?**
- when entering `show state`, the "Block-cache length" only displays zero
- this fix displays the actual value

- supports https://github.com/CityOfZion/neo-python/pull/930

**NOTE:** I am not sure what to do with the `BlockCacheCount` property of `LevelDBBlockchain`. It is still required for several functions like `PersistBlocks` but perhaps those will be deprecated?

**How did you solve this problem?**
trial and error

**How did you make sure your solution works?**
manual testing

**Are there any special changes in the code that we should be aware of?**
no

**Please check the following, if applicable:**

- [x] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?

### CODE REVIEW

File: neo/Prompt/Commands/Show.py [hunk 1]
```diff
@@ -238,8 +239,11 @@ def execute(self, arguments=None):
             bpm = diff / mins
             tps = Blockchain.Default().TXProcessed / secs
 
+        syncmngr = SyncManager()
+        l_block_cache = syncmngr.block_cache_count
+
         out = "Progress: %s / %s\n" % (height, headers)
-        out += "Block-cache length %s\n" % Blockchain.Default().BlockCacheCount
+        out += "Block-cache length %s\n" % l_block_cache
```
Comment (ixje) [inline neo/Prompt/Commands/Show.py]: i think a property is a bit of overkill for just this function. We can get the info directly
```suggestion
        out += "Block-cache length %s\n" % len(syncmngr.block_cache)
```

**Comment by jseagrave21** [comment]
Not sure why Travis failed; `make test` passed locally. I can see that some tests need to be fixed based on weird `requests` logs but again, the build passes locally. Also I had a bunch of fun learning how to mock the `SyncManager` class.

**Comment by jseagrave21** [comment]
Figured it out: linting 🤦‍♂️

**Comment by jseagrave21** [comment]
@ixje all good now, build passes

**Comment by ixje** [comment]
I was hoping I could just change the target branch from `asyncio` to the latest `asyncio_alpha2`. Unfortunately that didn't work without a whole bunch of conflicts. I think re-creating (copy/paste changes onto the new branch) is going to be fastest. If you can do that then I can merge it in.

**Comment by jseagrave21** [comment]
Okay, I will do that after work today.

**Comment by jseagrave21** [comment]
Closed for https://github.com/ixje/neo-python/pull/9

**Pull request closed without merging**